.travis.yml

```diff
@@ -20,12 +20,11 @@ addons:
 env:
     global:
         # Set defaults to avoid repeating in most cases
-        - NUMPY_VERSION=1.9
+        - NUMPY_VERSION=1.10
```

Comment:
I would maybe change this to ``stable`` to avoid having to update the default in future